pipelines/asana_dlt/const.py

```diff
@@ -0,0 +1,142 @@
+# Default start date for Asana API requests, only tasks started after this date will be collected
+DEFAULT_START_DATE = "2010-01-01T00:00:00.000Z"
+
+# Asana API request timeout
+TIMEOUT = 300
+
+# list of workspace fields to be retrieved from Asana API
+WORKSPACE_FIELDS = ["gid", "name", "is_organization", "resource_type", "email_domains"]
+
+# List of project fields to be retrieved from Asana API
+PROJECT_FIELDS = [
+    "name",
+    "gid",
+    "owner",
+    "current_status",
+    "custom_fields",
+    "default_view",
+    "due_date",
+    "due_on",
+    "is_template",
+    "created_at",
+    "modified_at",
+    "start_on",
+    "archived",
+    "public",
+    "members",
+    "followers",
+    "color",
+    "notes",
+    "icon",
+    "permalink_url",
+    "workspace",
+    "team",
+    "resource_type",
+    "current_status_update",
+    "custom_field_settings",
+    "completed",
+    "completed_at",
+    "completed_by",
+    "created_from_template",
+    "project_brief",
+]
+
+# List of section fields to be retrieved from Asana API
+SECTION_FIELDS = [
+    "gid",
+    "resource_type",
+    "name",
+    "created_at",
+    "project",
+    "projects",
+]
+
+# List of tag fields to be retrieved from Asana API
+TAG_FIELDS = [
+    "gid",
+    "resource_type",
+    "created_at",
+    "followers",
+    "name",
+    "color",
+    "notes",
+    "permalink_url",
+    "workspace",
+]
+
+# List of task fields to be retrieved from Asana API
+TASK_FIELDS = [
```

Comment:
better to use tuples becaue you can make them defaults in the function signatures